algorithms.py

```diff
@@ -266,7 +277,7 @@ def __init__(
             for k in range(sample_size):
                 j = len(self.nodes)
                 x = self.alloc_segment(0, self.m, j, pop_index)
-                self.L.set_value(x.index, self.m - 1)
+                # self.L.set_value(x.index, self.m - 1)
```

Comment:
It looks like this will break the code for the standard coalescent case. We need to make sure both code paths are working here before we merge as other people are developing algorithms using this file.